src/components/Profile.js

```diff
@@ -1,4 +1,16 @@
+/* eslint-disable no-undef */
 import React, {useState, useEffect} from "react";
+import Login from "./Login";
```

Comment:
Separating your own implemented file from npm package to another line.

Reply:
modified (other components as well)